entries/m/migros.ch.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Migros": {
+    "passwordless": "allowed",
+    "mfa": "allowed",
```

Comment:
```suggestion
```

From the documentation, it does not seem like Migros supports MFA with FIDO2. Can you confirm?

Reply:
On the first step, we implemented passkeys as a MFA. You can still use your old password to access your account on the SFA level. You gain your MFA status with the passkey.

Reply:
That's not noted in the documentation though.